### PR TITLE
Add solid_reactive effects for MIIIW BlackIO83

### PR DESCRIPTION
**Pull request by ArthurCyy**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by dunk2k — APPROVED** [review]

**Comment by zvecr** [comment]
> keyboard updates and refactors (eg. to data driven) must go through `develop`

The RGB changes would fall into this category.

For the matrix changes, why are such drastic changes being submitted now? This would highlight that there was no testing done on the original PR.

**Comment by ArthurCyy** [comment]
> > keyboard updates and refactors (eg. to data driven) must go through `develop`
> 
> The RGB changes would fall into this category.
> 
> For the matrix changes, why are such drastic changes being submitted now? This would highlight that there was no testing done on the original PR.

Sorry, I didn't add effects such as solid_reactive before causing me not to notice the problem, which is only visible after applying the solid_reactive effect.

**Comment by ArthurCyy** [comment]
Please review this PR, thanks!

**Review by drashna — APPROVED** [review]

**Review by dunk2k — APPROVED** [review]

**Comment by dunk2k** [comment]
branch has conflicts 🙈

**Comment by ArthurCyy** [comment]
> branch has conflicts 🙈

Solved.

File: keyboards/miiiw/blackio83/config.h [hunk 1]
```diff
@@ -18,39 +18,3 @@
...
-#    define ENABLE_RGB_MATRIX_RAINBOW_PINWHEELS
-#    define ENABLE_RGB_MATRIX_RAINDROPS
-#    define ENABLE_RGB_MATRIX_JELLYBEAN_RAINDROPS
-#    define ENABLE_RGB_MATRIX_HUE_BREATHING
-#    define ENABLE_RGB_MATRIX_HUE_PENDULUM
-#    define ENABLE_RGB_MATRIX_HUE_WAVE
-#    define ENABLE_RGB_MATRIX_PIXEL_RAIN
-#    define ENABLE_RGB_MATRIX_PIXEL_FLOW
-#    define ENABLE_RGB_MATRIX_PIXEL_FRACTAL
-#define RGB_MATRIX_DEFAULT_MODE RGB_MATRIX_SOLID_COLOR
```
[the remark this answers is not in the document]
Comment (ArthurCyy) [reply keyboards/miiiw/blackio83/config.h]: It has moved to info.json.

**Comment by zvecr** [comment]
Per the PR checklist,

> keyboard updates and refactors (eg. to data driven) must go through develop

**Review by dunk2k — APPROVED** [review]

**Review by waffle87 — APPROVED** [review]

**Pull request merged**